evals/registry/data/check_palindrome/check_palindrome.jsonl

```diff
@@ -0,0 +1,100 @@
+{"input": [{"role": "system", "content": "Evaluate if it is true that the given string is a palindrome."}, {"role": "user", "content": "tacocat"}], "ideal": "Yes"}
```

Comment:
Thank you for opening this PR! 

Please move any data inside `registry/data` to using git-lfs. We outline some instructions on how to install that dependency [here](https://github.com/openai/evals#downloading-evals).